### ISSUE TITLE
Train/valid split in examples

### ISSUE THREAD

**Issue by lorenzo-park** [thuml/Transfer-Learning-Library#28]
Hello,

Thank you for providing clean and nice implmentations. I am wondering that how the train and valid split is done in your example codes. It seems domainnet dataset has split option, but others don't so that the validation is done with training set?

Sincerely,
lorenzo-park

**Comment by JunguangJiang** [comment]
Our implementation follows the existing paper. In most DA classification datasets, such as Office-31 and Office-Home, the training set and test set are often the same. Since we cannot get the label on the target domain, this method will not cause overfitting.
